tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.psm1

```diff
@@ -1055,3 +1158,119 @@ function Set-BootstrapRepo
 	param([string]$Repo)
 	$script:BootStrapRepo = $Repo
 }
+
+<#
+.ExternalHelp help\AzureRM.Bootstrapper-help.xml 
+#>
+function Set-AzureRmDefaultProfile
+{
+  [CmdletBinding(SupportsShouldProcess = $true)]
+  param()
+  DynamicParam
+  {
+    $params = New-Object -Type System.Management.Automation.RuntimeDefinedParameterDictionary
+    Add-ProfileParam $params
+    Add-ForceParam $params
+    Add-ScopeParam $params
+    return $params
+  }
+  PROCESS {
+    $armProfile = $PSBoundParameters.Profile
+    $Scope = $PSBoundParameters.Scope
+    $Force = $PSBoundParameters.Force
+
+    $defaultProfile = $Global:PSDefaultParameterValues["*-AzureRmProfile:Profile"]
+    if ($defaultProfile -ne $armProfile)
+    {
+      if ($PSCmdlet.ShouldProcess("$armProfile", "Set Default Profile")) 
+      {
+        if (($Force.IsPresent -or $PSCmdlet.ShouldContinue("Are you sure you would like to set $armProfile as Default Profile?", "Setting $armProfile as Default profile")))
+        {
+          # Check Profile existence and choose proper profile
+          $profilePath = Select-Profile -Scope $Scope
+
+          # Set DefaultProfile for this session
+          $Global:PSDefaultParameterValues["*-AzureRmProfile:Profile"]="$armProfile"
+          $Global:PSDefaultParameterValues["Import-Module:RequiredVersion"]={ Get-ModuleVersion -armProfile $Global:PSDefaultParameterValues["*-AzureRmProfile:Profile"] -invocationLine $MyInvocation.Line }
+
+          # Edit the profile content
+          $profileContent = @"
+`$PSDefaultParameterValues["*-AzureRmProfile:Profile"]="$armProfile" `r`n
```

Comment:
See the comment above, I think that if we put content markers before an after the profile script, this will make it easier to detect and remove.